api/src/main/java/com/cloud/vm/VirtualMachine.java

```diff
@@ -52,7 +52,7 @@
         Migrating(true, "VM is being migrated.  host id holds to from host"),
         Error(false, "VM is in error"),
         Unknown(false, "VM state is unknown."),
-        Shutdowned(false, "VM is shutdowned from inside");
+        Shutdown(false, "VM is shutdown from inside");
```

Comment:
@GabrielBrascher Add database upgrade path to update any VMs in the old state to the new state name. (just a rare edge case).

Reply:
@rhtyd added a database upgrade schema from 4.13.0.0 to 4.13.1.0 upgrading VMs from state `Shutdowned` to `Shutdown`.

Reply:
@rhtyd does it look like a 4.13.1 fix? I had mapped it to 4.13.1.0, but I can target it to master/4.14.0.0; not really a bug fix, more likely an enhancement.